lib/util/uuid.c

```diff
@@ -50,7 +50,7 @@ struct uuid {
  * As per RFC 4122 section 4.4.
  */
 void
-sudo_uuid_create_v1(unsigned char uuid_out[16])
+sudo_uuid_create_v1(unsigned char uuid_out[restrict 16])
```

Comment:
I don't think the restrict keyword does anything here as there is only a single pointer.

Reply:
It's more for consistency.

Reply:
> I don't think the restrict keyword does anything here as there is only a single pointer.

I removed it just in case, however.